src/LodeRunner.UI/src/services/utilities.js

```diff
@@ -61,20 +63,27 @@ const getApi = async (endpoint) => {
 };
 
 const writeApi = (method, endpoint) => async (payload) => {
+  const resourceId = method === "PUT" ? `/${payload[TEST_RUN.id]}` : "";
```

Comment:
any way to use generic id and instead handle extraction in the test run service?

Reply:
this method is being using by writeConfig in config service. PUTS from there would fail under this change